src/credentials/status/reverse-sparse-merkle-tree.ts

```diff
@@ -130,8 +130,8 @@ export class RHSResolver implements CredentialStatusResolver {
   ): Promise<RevocationStatus> {
     try {
       return await this.getStatus(credentialStatus, opts);
-    } catch (e) {
-      const errMsg = e.reason ?? e.message;
```

Comment:
we have to leave `reason` here, because contract will return error if any inside reason field and next if depends on message